client/dashboard/components/guided-tour/step/index.tsx

```diff
@@ -117,9 +118,7 @@ export function GuidedTourStep( {
 				<VStack spacing={ 6 }>
 					<VStack spacing={ 2 }>
 						<HStack justify="space-between">
-							<Text as="h2" size={ 15 } weight={ 500 } lineHeight="20px">
-								{ currentTour.title }
-							</Text>
+							<SectionHeader title={ currentTour.title } level={ 3 } />
```

Comment:
This is done to make the "padding top" larger to match Figma:

|Before|After|
|-|-|
|<img width="357" height="163" alt="Image" src="https://github.com/user-attachments/assets/aaf8fba3-33e4-4fdd-ae2c-befa3fa3268c" />|<img width="354" height="173" alt="Image" src="https://github.com/user-attachments/assets/c87d8720-d0f5-47bd-b035-c5192885d36a" />